### ISSUE TITLE
preview file issue with removing file

### ISSUE THREAD

**Issue by DarkFisk** [vugar005/ngx-awesome-uploader#110]
````Hi, thanks for the great library.
I'm trying to setup it in my form, but when I'm editing, I need to load previously upload files.
But for things like removing, I need ID of the file, since `FilePreviewModel` have optional `uploadResponse` field, (where things like ID have to be logically), so I extended your Demo example by adding `uploadResponse` object.
```javascript
public ngOnInit(): void {
  setTimeout(() => {
    const files = [
      {
        fileName: 'My File 1 for edit.png', file: null, uploadResponse: { id: 1, filename: "My File 1 for edit.png" }
      },
      {
        fileName: 'My File 2 for edit.xlsx', file: null, uploadResponse: { id: 2, filename: "My File 2 for edit.xlsx" }
      }
    ] as FilePreviewModel[];
    this.uploader.setFiles(files);
  }, 1000);
  }
```

but when I clicked remove icon, `removeFile` of adapter from demo receive empty `uploadResponse` field.. I tried to investigate and found that you fill this field only for normally uploaded files, but for files added dynamically using `uploader.setFiles` method you override this field when merging it:
```javascript
public onRemove(fileItem: FilePreviewModel): void {
    this._uploadUnsubscribe();
    this.removeFile.next({
      ...fileItem,
      uploadResponse: this.uploadResponse
    });
   }
```

is it bug or I'm duing something wrong?

**Comment by DarkFisk** [comment]
@vugar005 what do you think?

**Comment by vugar005** [comment]
@DarkFisk  Thanks for PR. Fix, available in v15.0.1
